rope/base/worder.py

```diff
@@ -55,8 +55,8 @@ def get_word_at(self, offset):
     def get_primary_range(self, offset):
         return self._context_call("get_primary_range", offset)
 
-    def get_splitted_primary_before(self, offset):
-        return self._context_call("get_splitted_primary_before", offset)
+    def get_split_primary_before(self, offset):
+        return self._context_call("get_split_primary_before", offset)
```

Comment:
```suggestion
    def get_splitted_primary_before(self, offset):
        return self._context_call("get_splitted_primary_before", offset)
```

This is public interface, so shouldn't be fixed.